supports/integration-test/pyarrow_create_file.py

```diff
@@ -0,0 +1,45 @@
+import sys
+from multiprocessing import Pool
+import pyarrow as pa
+import time
+
+from util import *
+
+DIR_NAME = "pyarrow/"
+
+def create_file_pyarrow(ip,port,start,end):
+    fs = pa.hdfs.connect(ip,port)
+    fs.mkdir(TEST_DIR+DIR_NAME)
```

Comment:
Use TEST_DIR + random as dir name. We don't need to distinguish which dir is generated by pyarrow.